errbot/__init__.py

```diff
@@ -346,7 +346,10 @@ def wrapper(self, msg, args):
                 # Same happens with quotations marks, which are required for parsing
                 # complex strings in arguments
                 try:
-                    args = shlex.split(args.replace('—', '--').replace('“', '"').replace('”', '"'))
+                    #Map of strings to replace with sanitized versions
+                    sanitizers = {'—': '--', '“': '"', '”': '"'}
```

Comment:
Can you place this list near the top of the file, like: `ARG_BOTCMD_CHARACTER_REPLACEMENTS = {...}`? That will make it easier to locate next time anyone wants to update this list.

(Don't forget to move the comment block above as well)